pdc/apps/component/serializers.py

```diff
@@ -373,32 +373,6 @@ def to_internal_value(self, data):
             data['global_component'] = self.instance.global_component
         return super(ReleaseComponentSerializer, self).to_internal_value(data)
 
-    def validate_release(self, value):
-        if not isinstance(value, Release):
-            if isinstance(value, dict):
-                release_id = value['release_id']
-            else:
-                release_id = value
-            if release_id is None or release_id.strip() == "":
-                self._errors = {'release': 'This field is required.'}
-                return
-            release = get_object_or_404(Release, release_id=release_id)
-            if not release.is_active():
```

Comment:
@xychu That's a good point. However, the validation still needs to be updated, because the `value` argument is always an instance of `Release`. Raising `ValidationError` would also be a preferable way of failing.

Please note that for bulk update, the key is actually called `releases`, so this is quite confusing.